examples/log_to_file/src/main.rs

```diff
@@ -0,0 +1,52 @@
+// cargo install cargo-edit
+// cargo add log4rs log
```

Comment:
I'm not sure we want to be adding cargo plugins here.

Reply:
Removed!